src/test/java/seedu/address/model/person/AddressTest.java

```diff
@@ -10,27 +10,27 @@
 
     @Test
     public void constructor_null_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> new Address(null));
+        assertThrows(NullPointerException.class, () -> new Id(null));
     }
 
     @Test
     public void constructor_invalidAddress_throwsIllegalArgumentException() {
         String invalidAddress = "";
-        assertThrows(IllegalArgumentException.class, () -> new Address(invalidAddress));
+        assertThrows(IllegalArgumentException.class, () -> new Id(invalidAddress));
     }
 
     @Test
     public void isValidAddress() {
         // null address
-        assertThrows(NullPointerException.class, () -> Address.isValidAddress(null));
+        assertThrows(NullPointerException.class, () -> Id.isValidId(null));
 
         // invalid addresses
-        assertFalse(Address.isValidAddress("")); // empty string
-        assertFalse(Address.isValidAddress(" ")); // spaces only
+        assertFalse(Id.isValidId("")); // empty string
+        assertFalse(Id.isValidId(" ")); // spaces only
```

Comment:
In addition, to these 2 tests, we can try to assert for id longer than the usual length or doesn't contain the correct format.

For example,
```assertFalse(Id.isValidId("A0123456789X"));```
```assertFalse(Id.isValidId("0123456789"));```